samples/xamarin-forms/Cinephile/iOS/Cinephile.iOS.csproj

```
@@ -37,8 +39,10 @@
     <WarningLevel>4</WarningLevel>
     <MtouchArch>ARM64</MtouchArch>
     <ConsolePause>false</ConsolePause>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>iPhone Developer: Giusepe Casagrande (EL633D56TW)</CodesignKey>
```

This line should be  rolled back as others don't have your certificate.

```suggestion
    <CodesignKey>iPhone Developer</CodesignKey>
```